src/GovUk.Education.ExploreEducationStatistics.Notifier/README.md

```diff
@@ -0,0 +1,44 @@
+# Notifier Functions
+
+This project contains Azure Functions responsible for notifying users of changes to their subscribed publications using the Gov.UK Notify service.
+
+## Local Development and Testing
+
+Create a file in the root of the project directory called "local.settings.json" with the following structure:
+
+```
+{
+  "IsEncrypted": false,
+  "Values": {
+    "AzureWebJobsStorage": "UseDevelopmentStorage=true;DevelopmentStorageProxyUri=http://data-storage",
+    "FUNCTIONS_WORKER_RUNTIME": "dotnet",
+    "NotifyApiKey": "change-me",
+    "VerificationEmailTemplateId": "change-me",
+    "ReleaseEmailTemplateId": "change-me",
+    "ReleaseAmendmentEmailTemplateId": "change-me",
+    "ReleaseSupersededSubscribersEmailTemplateId": "change-me",
+    "ReleaseAmendmentSupersededSubscribersEmailTemplateId": "change-me",
+    "SubscriptionConfirmationEmailTemplateId": "change-me",
+    "TokenSecretKey": "change-me",
+    "BaseUrl": "http://localhost:7073/api/publication/",
+    "WebApplicationBaseUrl": "http://localhost:3000/",
+    "TableStorageConnString": "AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;DefaultEndpointsProtocol=http;QueueEndpoint=http://data-storage:10001/devstoreaccount1;TableEndpoint=http://data-storage:10002/devstoreaccount1;"
+  },
+  "ConnectionStrings": {
+  },
+  "Host": {
+    "CORS": "*"
+  }
+}
+
+```
+
+*This file should be automatically ignored by source control as it will contain sensitive information.*
```

Comment:
See comment on the deleted file about this potentially existing as an `.example` file instead of within the README to make it easier to maintain and copy.